content/tutorials/create-users-and-teams.md

```diff
@@ -88,24 +90,22 @@ In this exercise, you'll assign the users to their corresponding team:
 | almaz.russom      | Engineering |
 | yuan.yang         | Engineering |
 
-#### Create a team:
-
 Repeat the following steps for each team:
 
 1. In the sidebar, click **Configuration** -> **Teams**.
-1. Click **New team**.
-1. In the **Name** box, enter the name of the team.
-1. Click **Create**.
+2. Click **New team**.
+3. In the **Name** box, type the name of the team.
```

Comment:
Now I'm confused. Why type rather than enter here?

Also, I believe the style guide is encouraging lazy numbering?